otelcollector/metricextension/me_ds_internal_win.config

```diff
@@ -23,5 +23,7 @@
    "maxStringInternCacheSizeMb":5000,
    "interningSwapPeriodInMin":10000,
    "internalQueueSizeManagementPeriodInSec":10000,
-   "proxyDefinitionMode":1
+   "proxyDefinitionMode":1,
+   "memoryPoolReleasePeriodInMin":10,
+   "memoryPoolReleaseTriggerPercentage": 70
```

Comment:
i see both time & percentage. so whatever comes first it triggers deallocation ? Also 70% is when the mem usage becomes 70% from the highest usage ?

Reply:
Basically, this means that the memory release evaluation is performed every 10 minutes. 
Yes your explanation for percentage is correct.